.gitignore

```diff
@@ -26,12 +24,10 @@ chsdi/static/doc/build/
 chsdi/static/css/extended.min.css
 chsdi/static/css/blueimp-gallery.min.css
 chsdi/static/info.json
-deploy/hooks/pre-create-code
 node_modules
 MANIFEST
 buildout_branch.cfg
 25-mf-chsdi3.conf
-deploy/deploy-branch.cfg
 deploy/conf/00-branch.conf
```

Comment:
Should this not also be removed ?

Reply:
Below should we also not remove `rancher-*.yml` and what about `tests/varnish*` ?